versions/3.0.md

```diff
@@ -2365,42 +2370,147 @@ for the value that it is accompanying.  Tooling implementations MAY choose to
 validate compatibility automatically, and reject the example value(s) if they 
 are not compatible.
 
+Here is a single `example` in a model [Schema Object](#schemaObject):
```

Comment:
Minor nit, is _"single"_ necessary here?  Basically, you're saying _"Here is an `example` within a model"_.  Thoughts?